javascript-if/main.js

```diff
@@ -0,0 +1,75 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro
+ */
+
+function isUnderFive(number) {
+  if (number < 5) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function isEven(number) {
+  if (number % 2 === 0) {
+    return true;
+  }
+  return false;
+}
+
+function startsWithJ(string) {
+  if (string[0] === 'j' || string[0] === 'J') {
+    return true;
+  }
+  return false;
+}
+
+function isOldEnoughToDrink(person) {
+  if (person.age <= 21) {
+    return false;
+  } else {
+    return true;
+  }
+}
+
+function isOldEnoughToDrive(person) {
+  if (person.age >= 16) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function isOldEnoughToDrinkAndDrive(person) {
+  return false;
+}
+
+function categorizeAcidity(pH) {
+  if (pH === 7) {
+    return 'neutral';
+  } else
+  if ((pH <= 7) && (pH >= 0)) {
+    return 'acid';
+  } else
+  if (pH <= 14 && pH >= 7) {
+    return 'base';
+  } else {
```

Comment:
For clarity, you may want to remove the "equals to `7`" part of the acid and base comparisons.

Reply:
I definitely agree :)